### PR TITLE
README + fix for populate command

### PR DESCRIPTION
**Pull request by shmuelko**
fixed README
+ fix for populate command - it used old version changes to new version plus correct pull request for tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,53 +3,47 @@
 You will need to use 'create-testset' action once and 'populate-testset' action every time there is a new report.
 If the structure of the report changes (new tests added for example), you will need to use 'create-testset' action again. New TestSet will be created, but existing tests will be reused.
 
+If you don't have exsiting CONFIG_FILE and you want to use it go to https://firecracker-ui-prod.practitest.com/ and generate one.
+To login you can use your PractiTest credentials, follow the instructions in the link and then you can continue here, set config-path to your configuration file path.
 ## Usage
 
+### help
+
+At evrey point you can run:
```
Comment (stask) [inline README.md]: typo: every, not evrey

File: README.md [hunk 2]
```diff
@@ -3,53 +3,47 @@
...
-    --author-id=PRACTITEST_USER_ID \
-    --additional-test-fields '{"custom-fields": {"---f-123": "foo", "---f-124": "bar"}}' \
-    --additional-testset-fields '{"custom-fields": {"---f-125": "baz"}}' \
-    create-testset
-
-```
+You can set various custom fields for tests when they are created (especially useful if you have mandatory fields configured in your tests).
 
-Replace the field ids with the actual IDs, you can see all your custom fields by calling this API call: [get-all-custom-fields-in-your-project](https://www.practitest.com/api-v2/#get-all-custom-fields-in-your-project)
+Those configuration are set in config file to edit the config file go to https://firecracker-ui-prod.practitest.com/ and then use the example above again with the new generated config file.
```
Comment (stask) [inline README.md]: Don't think it's valid english.

File: src/practitest_firecracker/practitest.clj [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 ;; ===========================================================================
 ;; api version
-(def ^:const fc-version "1.4.0")
+(def ^:const fc-version "0.5.1")
```
Comment (stask) [inline src/practitest_firecracker/practitest.clj]: Did we already release 1.4.0? If so, we cannot go back and should always increase the version number.

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/practitest.clj (hunk 3 above)
Comment (shmuelko) [reply src/practitest_firecracker/practitest.clj]: If the version corresponds to release than no.

File: README.md (hunk 2 above)
Comment (shmuelko) [reply README.md]: Sorry, I got used to Grammarly to auto-correct me, will rewrite.

File: README.md (hunk 1 above)
Comment (shmuelko) [reply README.md]: will fix.

**Review by stask — APPROVED** [review]

**Pull request merged**